package.json

```diff
@@ -66,6 +63,11 @@
       "scss/at-mixin-pattern": "^(terra-)[a-z]+([a-z0-9-]+[a-z0-9]+)?$"
     }
   },
+  "files": [
```

Comment:
Can the .npmignore file be removed with these changes or is it still necessary?

Reply:
That makes sense. I'll remove it and double check by running the pack command.

Reply:
It doesn't appear to be needed